packages/flutter/lib/src/material/material.dart

```diff
@@ -87,13 +89,15 @@ abstract class MaterialInkController {
 ///
 /// The Material widget is responsible for:
 ///
-/// 1. Clipping: Material clips its widget sub-tree to the shape specified by
-///    [shape], [type], and [borderRadius].
-/// 2. Elevation: Material elevates its widget sub-tree on the Z axis by
+/// 1. Elevation: Material elevates its widget sub-tree on the Z axis by
 ///    [elevation] pixels, and draws the appropriate shadow.
-/// 3. Ink effects: Material shows ink effects implemented by [InkFeature]s
+/// 2. Ink effects: Material shows ink effects implemented by [InkFeature]s
 ///    like [InkSplash] and [InkHighlight] below its children.
 ///
+/// By default, [clip] is [Clip.none] and no clipping is performed. You
```

Comment:
when writing docs, avoid referring to the reader ("You may..."). The documentation uses the passive voice, because we don't know the context of the reader. For example, maybe the reader is trying to understand existing code they can't change (in which case, they can't override it).

Reply:
Done.